judgments/utils.py

```diff
@@ -212,4 +212,4 @@ def get_document_by_uri(document_uri: str) -> Judgment:
     )
 
     # raises a JudgmentNotFoundError if it doesn't exist
```

Comment:
It now raises `DocumentNotFoundError`.

```suggestion
    # raises a DocumentNotFoundError if it doesn't exist
```